src/main/java/com/google/sps/servlets/EventsServlet.java

```diff
@@ -211,4 +259,47 @@ public String sanitizeInput(String input) throws java.io.UnsupportedEncodingExce
       input = new String( input.getBytes("UTF-8") , 0, Math.min(MAX_STRING_BYTES, input.length()), "UTF-8");
       return input;
   }
+
+  public long getEventId() {
```

Comment:
Does this method just return the most recent event added to the database?

Reply:
It tracks how many events have been made in order to sequentially ID the events (first event will get an id of 0, etc)

Reply:
If you're looking for a unique ID to key events by, `datastore.put(eventEntity)` on line 225 will assign the new eventEntity a Key which has a unique ID which you can use.

By relying on datastore to provide unique IDs for each entity, you can access the unique event id by:
```
datastore.put(newEventEntity);
long myEventUniqueId = newEventEntity.getKey().getId();
```